client/rpc/client_test.go

```diff
@@ -147,7 +147,7 @@ func testPermissions(t *testing.T,
 	nonceString := ""
 
 	_, err := Permissions(nodeClient, keyClient, publicKeyString, addressString,
-		nonceString, "set_base", []string{permAddressString, "root", "true"})
+		nonceString, "setBase", []string{permAddressString, "root", "true"})
```

Comment:
That's fine, but working directly on the EVM/SNative is not the same as going through the RPC (as we've seen with https://github.com/eris-ltd/eris-db/pull/477). I would recommend that there is atleast one of every RPC permission call type included in here and to ensure that they are passing.